core/src/services/memcached/backend.rs

```diff
@@ -150,6 +175,9 @@ impl Builder for MemcachedBuilder {
             endpoint,
             username: self.config.username.clone(),
             password: self.config.password.clone(),
+            tls: self.config.tls.clone(),
+            cafile: self.config.cafile.clone(),
+            host,
```

Comment:
Why we need a seperate `host`?

Reply:
Because  `ServerName` requires it, I don't want to return an additional error.